streaming_form_data/parser.py

```diff
@@ -20,12 +20,13 @@ def parse_content_boundary(headers: Mapping[str, str]) -> bytes:
     if not content_type:
         raise ParseFailedException('Missing Content-Type header')
 
-    value, params = cgi.parse_header(content_type)
+    m = EmailMessage()
```

Comment:
Let's make this variable name a bit more descriptive. Perhaps something like `message`?

Reply:
Done in 188232e, thanks!